### PR TITLE
Revert CH back to September

### PR DESCRIPTION
**Pull request by SwiftySalmon**
Made a big mess, revert CH back to September, will fix in January

### CODE REVIEW

File: R/process_sc_all_care_home.R [hunk 1]
```diff
@@ -29,54 +29,44 @@ process_sc_all_care_home <- function(
...
-    dplyr::mutate(ch_discharge_date = fix_sc_end_dates(
-      .data$ch_admission_date,
-      .data$ch_discharge_date,
-      .data$period
-    )) %>%
+    ) %>%
     dplyr::left_join(sc_demog_lookup,
       by = c("sending_location", "social_care_id")
     ) %>%
     replace_sc_id_with_latest()
```
Comment (rchlv) [inline R/process_sc_all_care_home.R:52]: Thank you for updating the code! I'm not sure if my point is relevant so feel free to leave it!

I noticed in the resolved PR "Deal with NA in ch_provider and missing group_by()", this function is used later in the script (found on lines 89 and 90) and passes fixed_ch_provider through this function to create a new object fixed_sc_id. fixed_sc_id then feeds into fixed_nursing_provision

File: R/process_sc_all_care_home.R [hunk 2]
```diff
@@ -99,47 +89,58 @@ process_sc_all_care_home <- function(
...
+        .data[["ch_provider"]]
+      )
     ) %>%
+    dplyr::select(
+      -"min_ch_provider",
+      -"max_ch_provider"
+    ) %>%
+    # tidy up ch_provider using 6 when disagreeing values
+    tidyr::fill(.data[["ch_provider"]], .direction = "downup") %>%
+    dplyr::ungroup()
```
Comment (Jennit07) [inline R/process_sc_all_care_home.R:134]: Comparing to master, these lines are duplicate code and the same thing is done on lines 60-88

File: R/process_sc_all_care_home.R [hunk 3]
```diff
@@ -99,47 +89,58 @@ process_sc_all_care_home <- function(
...
       .data[["ch_admission_date"]]
     ) %>%
     # fill in nursing care provision when missing
     # but present in the following entry
     dplyr::mutate(
       nursing_care_provision = dplyr::na_if(.data[["nursing_care_provision"]], 9L)
     ) %>%
-    tidyr::fill(all_of("nursing_care_provision"), .direction = "downup") %>%
-    dplyr::ungroup()
+    tidyr::fill(.data[["nursing_care_provision"]], .direction = "downup")
```
Comment (Jennit07) [inline R/process_sc_all_care_home.R:102]: On master there is a line we are missing after here and also need the ungroup()

File: R/process_sc_all_care_home.R (hunk 1 above)
Comment (Jennit07) [reply R/process_sc_all_care_home.R:52]: Fixed this - we have reverted back to master copy

File: R/process_sc_all_care_home.R (hunk 2 above)
Comment (Jennit07) [reply R/process_sc_all_care_home.R]: reverted back to master copy

File: R/process_sc_all_care_home.R (hunk 3 above)
Comment (Jennit07) [reply R/process_sc_all_care_home.R:105]: reverted back to master

**Review by Jennit07 — APPROVED** [review]
Discussed this on a call and tested the files - looks in line with september update

**Pull request merged**